### PR TITLE
Multiple examples added

### PR DESCRIPTION
**Pull request by davidmanzanares**

### CODE REVIEW

**Comment by davidmanzanares** [comment]
I'm using that convention because it's easier to copy and paste the URLs into the browser.

File: example/styling/cielab.2.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Single layer | CARTO</title>
```
Comment (IagoLast) [inline example/styling/cielab.2.html]: wrong titles

Comment (davidmanzanares) [reply example/styling/cielab.2.html]: Yep, on all of them. However, for all the styling examples there is much copy-pasted boilerplate. We could use a JSON for all of these styling examples, right?

File: src/index.js [hunk 2]
```diff
@@ -9,12 +9,15 @@
...
 
 // Source namespace has dataset and SQL
 const source = { Dataset };
 
+
 export { Layer };
 export { source };
 export { Style };
 export { parseStyle }; // TODO(@iago): temporal export, we must find a better place
+export { expressions };
```
Comment (IagoLast) [inline src/index.js]: we want to export this as `carto.expressions` or we want `carto.style.<expression>` ? @davidmanzanares @Jesus89

Comment (Jesus89) [reply src/index.js]: I vote for `carto.style.expression` but we can talk about this in the `Expressions` epic.

**Pull request merged**